### DISCUSSION TITLE
I want to use ScrollMagic or smooth scrollbar in Nuxt3. Do you have any good examples?

### DISCUSSION THREAD

**Discussion by tendecade** [nuxt/nuxt#19997] [Questions]
### Describe the feature

"I originally used 'vue3 smooth scroll bar' in Vue3, but now I want to migrate to Nuxt3. After introducing this scroll bar, the page is not slideable.". Therefore, I need to ask if there are any relevant cases that can guide me to use this scroll bar in Nuxt3, or recommend smooth scrolling dependencies that can be used in Nuxt3. Thank you.

### Additional information

- [ ] Would you be willing to help implement this feature?
- [ ] Could this feature be implemented as a module?

### Final checks

- [X] Read the [contribution guide](https://nuxt.com/docs/community/contribution).
- [X] Check existing [discussions](https://github.com/nuxt/nuxt/discussions) and [issues](https://github.com/nuxt/nuxt/issues).

**Comment by mouldt** [comment]
does this help: https://youtu.be/KVk372L17N8
